### PR TITLE
Options refactor

### PR DESCRIPTION
**Pull request by stephengreenfield**
@NGPVAN/yellow Please code review

### CODE REVIEW

File: lib/direct_connect/direct_connect_gateway.rb [hunk 1]
```diff
@@ -18,57 +18,56 @@ class Gateway < ActiveMerchant::Billing::Gateway
 
       DIRECT_CONNECT_CODES = {
           0 => :success,
+          12 => :verificationRejected,
```
Comment (BaconSoap) [inline lib/direct_connect/direct_connect_gateway.rb]: we should standardize these and other variables (including the ones that existed already) to use `snake_case`

File: lib/direct_connect/direct_connect_gateway.rb [hunk 2]
```diff
@@ -129,165 +120,153 @@ def scrub(transcript)
...
+        post[:cardinfokey] = card_info_key
+        post[:ccaccountnum] = credit_card.number
+        post[:ccexpdate] = credit_card.expiry_date.expiration.strftime('%02m%02y')
+        post[:ccnameoncard] = credit_card.name
+        post[:ccstreet] = customer.address.street1
+        post[:cczip] = customer.address.zip
         post[:extdata] = nil
       end
 
-      def store_credit_card_info(post, options)
```
Comment (BaconSoap) [inline lib/direct_connect/direct_connect_gateway.rb]: we should name this to match the other ones that add data - like `add_store_credit_card_info`

**Comment by BaconSoap** [comment]
:cat2:

File: spec/direct_connect/test_helper.rb [hunk 3]
```diff
@@ -138,41 +138,43 @@ module Fixtures
...
-        address2: 'Apt 1',
-        company:  'Widgets Inc',
-        city:     'Ottawa',
-        state:    'ON',
-        zip:      'K1C2N6',
-        country:  'CA',
-        phone:    '(555)555-5555',
-        fax:      '(555)555-6666'
-      }.update(options)
+    def address
```
Comment (TechShaman) [inline spec/direct_connect/test_helper.rb]: the naming seems wrong here - address is returning a Customer?

Comment (afrieden) [inline spec/direct_connect/test_helper.rb]: the naming seems wrong for this - address is returning a Customer?

File: spec/direct_connect/remote/remote_direct_connect_gateway_test.rb [hunk 4]
```diff
@@ -8,15 +8,15 @@ class RemoteDirectConnectTest < MiniTest::Test
   def setup
     @gateway = KillBill::DirectConnect::Gateway.new(fixtures(:direct_connect))
 
-    @amount = 100
-    @credit_card = credit_card('4111111111111111')
-    @declined_card = credit_card('4111111111111112')
+    @customer = address
```
Comment (afrieden) [inline spec/direct_connect/remote/remote_direct_connect_gateway_test.rb]: this line is confusing.
address is returning a Customer and should be renamed

**Comment by afrieden** [comment]
Looks good :coffee:

**Pull request merged**